### DISCUSSION TITLE
Kitty command work over SSH

### DISCUSSION THREAD

**Discussion by jpmvferreira** [kovidgoyal/kitty#4137] [Q&A] [answered]
This might seem like it has an obvious answer, but when SSHing into a remote server, which I do not administer, and copying the terminfo with the appropriate kitten, all is well, however, I would like to be use specific kittens, for instance the icat, to see images in the terminal when connected over SSH, but kitty is obviously not available on the other side.

What should I have to do to have kittens available while in SSH?

**Comment by kovidgoyal** [comment, marked as answer]
Install kitty there, and no, you dont need to be root to install kitty.
https://sw.kovidgoyal.net/kitty/binary/ 

Or write your own script that uses the kitty graphics protocol to
display images: https://sw.kovidgoyal.net/kitty/graphics-protocol/

**Reply by jpmvferreira** [reply]
I thought of that, but wouldn't that require me to manually update each time I update kitty on my system, or is there some backwards compatibility?
And other reason why I didn't want to do so is because this machine is fairly old, currently running CentOS 6.6 if I'm not mistaken, so issues might arise.

But regardless, it's not like giving it a try will blow up the computer, so if really there is no other way (other than writing my own script with the kitty graphics protocol) I'm gonna go with it.

**Reply by kovidgoyal** [reply]
The graphics protocol  changing in backwards incompatible ways is *extremely* unlikely. And am not sure what you are asking. There needs to be *something* that speaks the protocol on the remote side, either a kitten or your own script or some third party program of which now-a-days there are dozens.

**Reply by jpmvferreira** [reply]
Yeah, the system doesn't have up to date glibc versions, and installing them through virtual environments will polute my other packages, I will be looking for one of those server-side third party programs to manage to display an image through SSH, since that is all I really need, at least for now.
Thanks!
